roku_dev_cli/roku_dev_cli.py

```diff
@@ -816,6 +820,8 @@ def main():
     parser.add_argument('--select-roku-ips', action='store_true', help="Filters list of IPs and returns all valid, reachable, roku IPs in list.")
     parser.add_argument('--select_roku_ips', action='store_true', help=argparse.SUPPRESS)
 
+    parser.add_argument('--package_all', action='store_true', help=argparse.SUPPRESS)
```

Comment:
Can you change the help string here to something like "Explicitly zip everything in working directory"